examples/python/v2/cli/create_did.py

```diff
@@ -1,11 +1,7 @@
 from sock import post
 
 
-def main():
-    endpoint = "/identifiers"
-    json_response = post(endpoint, {})
-    print(json_response)
+json_response = post("/identifiers")
 
-
-if __name__ == "__main__":
-    main()
+print("The response is as follows.\n")
+print(json_response)
```

Comment:
```suggestion
print(f"The response is as follows.\n {json_response}")
```
[imo]
You can use f-string 👍

Reply:
THX.

We would like to separate the output of comments from the output of values. I want this to be consistent for all examples.

Also, since we need two line breaks here, the code you suggested would change the output.

Therefore, we would like to keep them as they are.